crates/test_utils/src/connector_auth.rs

```diff
@@ -93,7 +93,7 @@ impl ConnectorAuthentication {
 }
 
 #[derive(Clone, Debug, Deserialize)]
-pub struct ConnectorAuthenticationMap(HashMap<String, ConnectorAuthType>);
+pub struct ConnectorAuthenticationMap(HashMap<String, CollectionAuthType>);
```

Comment:
Please revert these changes back, it has many dependencies and I'm pretty sure it will break UI and Webhook UI tests.